CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ endif()
 
 add_library(crossplatform_compiler_flags INTERFACE)
 target_compile_features(crossplatform_compiler_flags INTERFACE cxx_std_11)
-target_compile_options(crossplatform_compiler_flags INTERFACE -std=c++11 -Werror)
+target_compile_options(crossplatform_compiler_flags INTERFACE -std=c++11 -Werror -Wall)
```

Comment:
What about platform flags? Should we not enable both of these options there?

Reply:
Yeah, good point. Adding